packages/main/src/TimePicker.ts

```diff
@@ -12,20 +12,19 @@ import {
  * @class
  *
  * <h3 class="comment-api-title">Overview</h3>
- * The <code>ui5-time-picker</code> component provides an input field with assigned sliders which are opened on user action.
- * The <code>ui5-time-picker</code> allows users to select a localized time using touch,
- * mouse, or keyboard input. It consists of two parts: the time input field and the
- * sliders.
+ * The <code>ui5-time-picker</code> component provides an input field with assigned clocks which are opened on user action.
+ * The <code>ui5-time-picker</code> allows users to select a localized time using touch, mouse, or keyboard input.
+ * It consists of two parts: the time input field and the clocks.
  *
  * <h3>Usage</h3>
  * The user can enter a time by:
  * <ul>
- * <li>Using the sliders that are displayed in a popup</li>
+ * <li>Using the clocks that are displayed in a popup</li>
  * <li>Typing it in directly in the input field</li>
  * </ul>
  * <br><br>
- * When the user makes an entry and chooses the enter key, the sliders shows the corresponding time.
```

Comment:
When the user makes an entry and chooses the enter key, the sliders show the corresponding time.